15th april 2016/3.rb~

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/ruby -w
 
-arr=["hello", "its", "an", "array", "date", 2016]
+arr = ["hello", "its", "an", "array", "date", 2016]
```

Comment:
don't use double quotes, unless you are interpolating string.
also use ```%w(hello its an array date 2016)``` to define array